docs-markdown/markdownlint-custom-rules/securelinks.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+"use strict";
+
+const common = require("./common");
```

Comment:
We can drop common and detailStrings since they are not used in this rule.

Reply:
Lets not close this yet, I think I should move the href regex to common *and* consider if I should move any error text to strings file